pkg/unikernel/libvirt/connection.go

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Should add a build flag for unikernel runtime, it requires libvirt library for building

Reply:
Will do.